.travis.yml

```diff
@@ -13,57 +13,23 @@ services:
 
 jobs:
   include:
-  - stage: Tests
-    name: "minishift"
-    env: CPLATFORM=minishift TARGET=functests KUBECTL=oc OC=oc V=5 CI_CONFIG_FEATURE_GATES=DataVolumes,LiveMigration,CPUManager
-
   - stage: Build and Deploy
     name: Build and Deploy to GitHub
     if: tag IS present
     before_script: skip
     script: make release "IMAGE_TAG=$TRAVIS_TAG"
     deploy:
       provider: releases
-      api_key: "$GITHUB_TOKEN"
+      api_key: "$GITHUB_RELEASE_TOKEN"
```

Comment:
do you set correct variable in travis?

Reply:
Yes, https://travis-ci.com/github/kubevirt/kubevirt-ssp-operator/settings